languages/elixir/exercises/concept/processes/.docs/introduction.md

```diff
@@ -0,0 +1,37 @@
+In Elixir, all code runs inside processes.
```

Comment:
My compliments for how well you've explained things in this document! Really excellent!